SeleniumPracticalExercise/TestCases/AddEmployee.cs

```diff
@@ -1,31 +1,86 @@
 ﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.WaitHelpers;
 using SeleniumPracticalExercise.TestCases.Common;
+using SeleniumPracticalExercise.Common;
+using static SeleniumPracticalExercise.Common.Utils;
+using static SeleniumPracticalExercise.PageObjects.Common.BasePageLocal;
+using SeleniumPracticalExercise.PageObjects.Common;
+using System.Security.Cryptography.X509Certificates;
+using SeleniumPracticalExercise.PageObjects;
 
-namespace SeleniumPracticalExercise.TestCases
-{
-    class AddEmployee : BaseTestLocal
+class AddEmployee : BaseTestLocal
+
+    {
+    [Test]
+    [Category("Add Employee")]
+    public void AddEmployeeTest()
     {
-        [Test]
-        [Category("Add Employee")]
-        public void AddEmployeeTest()
-        {
-            // Steps to automate:
-            // 1. Navigate to https://opensource-demo.orangehrmlive.com/web/index.php/auth/login
-            // 2. Log in using Username: Admin, Password: admin123
-            // 3. Click "PIM" in the left nav
-            // 4. Click "+Add"
-            // 5. Randomly generate a first name (6 characters) and last name (8 characters) and enter them into the form
-            // 6. Get the Employee Id for use later
-            // 7. Click Save
-            // 8. Click "PIM" in the left nav
-            // 8. Search for the employee you just created by Employee Id
-            // 9. In the employee search results, use NUnit asserts to validate that Id, First Name, and Last Name are correct
-            //
-            // NOTE:
-            // - Use the provided WebDriver methods in BasePageLocal.cs
-            // - Create page objects as needed
-            // - Document all methods using XML documentation, https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/
-
-        }
+        var user = "Admin"; // declaring username for login
```

Comment:
In general, use comments sparingly. In this case, it should be obvious by looking at the code that you are storing a string in a variable. Save comments to describe code that isn't easy to understand, etc.